hack/markdownlint.sh

```diff
@@ -20,7 +20,7 @@ else
     --entrypoint sh \
     --workdir /workdir \
     docker.io/pipelinecomponents/markdownlint:0.12.0@sha256:0b8f9fcf0410257b2f3548f67ffe25934cfc9877a618b9f85afcf345a25804a2 \
-    /workdir/hack/markdownlint.sh "${@}"
+    /workdir/hack/markdownlint.sh "$@"
```

Comment:
FYI, this is Mac's old Bash3/zsh compatilibity issue where this syntax is not supported.